chain-signatures/node/src/cli.rs

```diff
@@ -63,6 +63,10 @@ pub enum Cli {
         /// referer header for mainnet whitelist
         #[arg(long, env("MPC_CLIENT_HEADER_REFERER"), default_value(None))]
         client_header_referer: Option<String>,
+        #[clap(flatten)]
+        mesh_options: mesh::Options,
+        #[clap(flatten)]
+        message_options: http_client::Options,
```

Comment:
hmm, both these should just be in the contract config, especially `message_options` since that is very similar to our generation timeout configuration